frontend/public/index.html

```diff
@@ -1,21 +1,22 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <!--
+<script src="https://apis.google.com/js/api.js" async defer></script>
+<meta name="google-signin-client_id" content="241748771473-0r3v31qcthi2kj09e5qk96mhsm5omrvr.apps.googleusercontent.com">
```

Comment:
Why is this here? We shouldn't expose the clientId to the public. Is this needed here for some reason? If so, we might have to figure out a different way to put this value here.

Reply:
If it has to appear on the client side, then there is no way to hide it. I think client id is designed to be public. It's the API key that has to be kept private.

Reply:
removed these lines, wasn't necessary to include them